### PR TITLE
Fix starship skills and proficiencies 

### PR DESCRIPTION
**Pull request by afwolfe**
Fixes the errors thrown when attempting to edit or roll starship skills or proficiencies on SW5e 2.2.2+ as reported in #728

### CODE REVIEW

File: module/applications/actor/proficiency-config.mjs [hunk 1]
```diff
@@ -14,7 +14,6 @@ import * as Trait from "../../documents/actor/trait.mjs";
  * @param {ProficiencyConfigOptions} options  Additional configuration options.
  */
 export default class ProficiencyConfig extends BaseConfigSheet {
-
```
Comment (afwolfe) [inline module/applications/actor/proficiency-config.mjs:17]: I did not directly modify these (and other) lines, but VS Code picked up on the .prettierrc in the root of the repo and applied them. Should this style be enforced elsewhere or should I revert these additional changes for now?

File: module/applications/actor/proficiency-config.mjs [hunk 2]
```diff
@@ -49,8 +48,12 @@ export default class ProficiencyConfig extends BaseConfigSheet {
 
   /** @inheritdoc */
   get title() {
-    const label = this.isSkill ? CONFIG.SW5E.skills[this.options.key].label : Trait.keyLabel("tool", this.options.key);
-    return `${game.i18n.format("SW5E.ProficiencyConfigureTitle", {label})}: ${this.document.name}`;
+    const label = this.isSkill
+      ? this.document.type === "starship"
```
Comment (afwolfe) [inline module/applications/actor/proficiency-config.mjs:52]: This is based on the `title()` method in [module/applications/actor/skill-config.mjs#L39](https://github.com/sw5e-foundry/sw5e/blob/develop/module/applications/actor/skill-config.mjs#L39)

File: static/templates/actors/newActor/starship-sheet.hbs [hunk 3]
```diff
@@ -164,9 +164,10 @@
           <h1>{{localize "SW5E.Skills" }}</h1>
           <ol>
             {{#each config.starshipSkills as |obj s|}} {{#with (lookup ../skills s) as |skill|}}
-            <li class="skill {{#if skill.value}}proficient{{/if}}" data-skill="{{s}}">
+            <li class="proficiency-row skill {{#if skill.value}}proficient{{/if}}" data-property="skills" data-key="{{s}}">
```
Comment (afwolfe) [inline static/templates/actors/newActor/starship-sheet.hbs:167]: This is based on the Skills section of [static/templates/actors/newActor/parts/swalt-core.hbs#L36](https://github.com/sw5e-foundry/sw5e/blob/develop/static/templates/actors/newActor/parts/swalt-core.hbs#L36)

File: module/applications/actor/proficiency-config.mjs (hunk 1 above)
Comment (Ikaguia) [reply module/applications/actor/proficiency-config.mjs:17]: Alright, so it seems like the `prettierrc` file is no longer used (and we will remove it), instead we are using `eslintrc` with `gulp lint`

File: module/applications/actor/proficiency-config.mjs (hunk 2 above)
Comment (Ikaguia) [reply module/applications/actor/proficiency-config.mjs:52]: This looks correct

File: static/templates/actors/newActor/starship-sheet.hbs (hunk 3 above)
Comment (Ikaguia) [reply static/templates/actors/newActor/starship-sheet.hbs:167]: This looks correct

**Review by Ikaguia — CHANGES_REQUESTED** [review]
The `prettierrc` changes should be reverted, but other than that everything seems fine

File: module/applications/actor/proficiency-config.mjs (hunk 1 above)
Comment (afwolfe) [reply module/applications/actor/proficiency-config.mjs]: I reverted these whitespace changes, and have made 3 separate commits and rebased my changes on top of them:
1. Remove .prettierrc
2. Run `npm run lint:fix` - there were a handful of unrelated fixes that needed applied
3. Fix starship skills and proficiencies

**Review by Ikaguia — APPROVED** [review]
Everything looks correct now

**Pull request merged**